### DISCUSSION TITLE
alternative completion-style for `cape-dict`

### DISCUSSION THREAD

**Discussion by fakeGenuis** [minad/cape#121] [Q&A] [answered]
Set `completion-styles` to `orderless` is really nice when complete commands, but for dict words it's counterintuitive. For example, type "re" and `cape-dict` will gives "abbrev" candidate first. Is there any variable override that behavior only for `cape-dict`?

**Comment by minad** [comment, marked as answer]
See completion-category-overrides for the category cape-dict.
